server/superdesk/utils.py

```diff
@@ -75,6 +75,17 @@ def is_hashed(input_str):
     return input_str.startswith('$2a$')
 
 
+def merge_dicts(dict_args):
```

Comment:
It would be slightly more pythonic if this is changed to `*dict_args` - this way the calling code won't have to explicitly pack the dictionariy arguments into a list or a tuple.